src/sanitizer.js

```diff
@@ -70,6 +80,15 @@ const WHITELISTED_FORMAT_TAGS = [
 
 
 /** @const {!Array<string>} */
+const WHITELISTED_ATTRS = [
+  'fallback',
+  'href',
+  'on',
+  'placeholder',
```

Comment:
This will be a limited list because these are only AMP attributes applied to non-AMP tags and there are only a few of those. AMP elements themselves are validated separately.